src/components/annotations/arrow-annotation.tsx

```diff
@@ -53,16 +53,17 @@ interface IArrowAnnotationProps {
   canEdit?: boolean;
   deleteArrow: (arrowId: string) => void;
   documentBottom: number;
+  documentLeft: number;
+  documentRight: number;
   documentTop: number;
```

Comment:
Might consider a single `documentBounds` prop with `left`, `top`, `right`, `bottom` props for organizational purposes at some point.